src/main/java/gregtech/common/metatileentities/electric/MetaTileEntityPump.java

```diff
@@ -154,30 +171,40 @@ private int getMaxPumpRange() {
     private boolean isStraightInPumpRange(BlockPos checkPos) {
         BlockPos pos = getPos();
         return checkPos.getX() == pos.getX() &&
-            checkPos.getZ() == pos.getZ() &&
-            pos.getY() < checkPos.getY() &&
-            pos.getY() + pumpHeadY >= checkPos.getY();
+                checkPos.getZ() == pos.getZ() &&
+                pos.getY() < checkPos.getY() &&
+                pos.getY() + pumpHeadY >= checkPos.getY();
     }
 
-    private void updateQueueState(int blocksToCheckAmount) {
-        BlockPos selfPos = getPos().down(pumpHeadY);
-
-        for(int i = 0; i < blocksToCheckAmount; i++) {
-            BlockPos checkPos = null;
-            int amountIterated = 0;
-            do {
-                if (checkPos != null) {
-                    blocksToCheck.push(checkPos);
-                    amountIterated++;
-                }
-                checkPos = blocksToCheck.poll();
+    private BlockPos getValidPos(Deque<BlockPos> posDeque) {
```

Comment:
The passed argument is unused in this method. Any reason for not removing it?

Reply:
That's a copy paste fail the arg should get used